### ISSUE TITLE
Better testing

### ISSUE THREAD

**Issue by anoopsarkar** [sfu-natlang/lensingwikipedia#212]
We should probably have some regression tests for this project. One way to enable this might be to use [gor](https://github.com/buger/gor)? Any thoughts?

**Comment by theq629** [comment]
Note that we do already have regression tests for the backend.

**Comment by vlad003** [comment]
I don't think we really have the kind of traffic/infrastructure to justify using gor.

I was actually thinking of using py.test to standardize and automate the query backend tests.

**Comment by anoopsarkar** [comment]
Let's leave this issue dormant for a while until we really need to add a testing suite.
